ppa/archive/import_util.py

```diff
@@ -292,7 +309,7 @@ def rsync_data(self):
                 subprocess.run(args=rsync_cmd.split(), check=True)
             except subprocess.CalledProcessError as err:
                 logger.error(
-                    "HathiTrust rsync failed — %s / command: %s"
+                    "HathiTrust rsync failed — %s / command: %s"
```

Comment:
Did this line actually change? Or, is it just the diff failing.

Reply:
there was a weird character here! I removed it but you can't tell what the change is, I guess it was a different kind of whitespace